commit 01c7e549b2b978e69d6509864d4d067c555cb81b
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sat Jan 6 22:19:13 2018 +0100

    hurd: Fix posix glob test
    
            * posix/tst-glob_symlinks.c [!PATH_MAX]: Define PATH_MAX macro.

diff --git a/posix/tst-glob_symlinks.c b/posix/tst-glob_symlinks.c
index 0f3478d3d0..df3baa8757 100644
--- a/posix/tst-glob_symlinks.c
+++ b/posix/tst-glob_symlinks.c
@@ -57,6 +57,9 @@ create_link (const char *base, const char *fname, char *linkname,
   add_temp_file (linkname);
 }
 
+#ifndef PATH_MAX
+# define PATH_MAX 1024
+#endif
 static char valid_link[PATH_MAX];
 static char dangling_link[PATH_MAX];
 static char dangling_dir[PATH_MAX];
